doc/getting_started/index.rst

```diff
@@ -346,23 +346,32 @@ including: compiler, assembler, linker, and their dependencies.
             chmod +x zephyr-sdk-0.11.2-setup.run
             ./zephyr-sdk-0.11.2-setup.run -- -d ~/zephyr-sdk-0.11.2
 
-      #. Set environment variables to let the build system know where to
-         find the toolchain programs:
+         .. note::
+	    It is recommended to install the Zephyr SDK at one of the following locations:
 
-         .. code-block:: bash
+            * ``$HOME/zephyr-sdk[-x.y.z]``
+            * ``$HOME/.local/zephyr-sdk[-x.y.z]``
+            * ``$HOME/.local/opt/zephyr-sdk[-x.y.z]``
+            * ``/opt/zephyr-sdk[-x.y.z]``
+            * ``/usr/zephyr-sdk[-x.y.z]``
+            * ``/usr/local/zephyr-sdk[-x.y.z]``
```

Comment:
Add `$HOME/bin/zephyr-sdk[-x.y.z]`

Reply:
We can add all the search folders we like, but question is if we should add this folder to recommended list.

Personally, I think the `bin` folder is for the executables (or links to executables) a not full install trees, like the SDK.
But i'm open to add any location user's find useful, as the Zephyr SDK cmake packages is about making live easier, then I would like to hear more thoughts on this.

Reply:
Well, I am in favor of adding it. It doesn't harm (perhaps a few ms wasted on looking for the package there) and it will benefit many? users.

Reply:
I'm not so fond of it, but i'm overruled by my own fondness of keeping our users happy.
so it's in.